src/main/pages/che-7/overview/ref_about_hosted_che.adoc

```diff
@@ -0,0 +1,50 @@
+[id="about_hosted_che_{context}"]
+= About Hosted Che
+
+Hosted{nbsp}Che is link:https://www.eclipse.org/che/[Eclipse{nbsp}Che] hosted by Red{nbsp}Hat.
+Che server is running on OpenShift{nbsp}Dedicated and user base is spread across multiple OpenShift{nbsp}Online{nbsp}clusters where workspaces are created.
+
+[id="terms-of-service_{context}"]
+== Terms of Service
+
+Hosted{nbsp}Che has the following usage limits and terms of service:
+
+Storage:: `1GB`
+
+RAM:: `3GB`
+
+CPU:: `6 Cores`
+
+Concurrent workspaces:: `1`
+
+Number of workspaces:: `Unlimited`
+
+Number of projects per workspace:: `Unlimited`
+
+Usage time limit:: `Unlimited`
+
+Maximum Account Inactivity period:: `30 days`
+
+[NOTE]
+====
+If a user does not login to Hosted{nbsp}Che for more than 30 days the account will be automatically deprovisioned and all existing workspaces will be lost.
+In order to start using Hosted{nbsp}Che again, a user will need to re-register.
+====
+
+Workspace Idling time:: `15 minutes`
+
+[NOTE] 
+====
+Once started workspace can be used for an unlimited period of time.
+However, if there are no interactions with IDE workspace will be gracefully stopped after 15 minutes.
+====
+
+Ephemeral mode:: defaults to `Active`
+
+[NOTE]
+====
+In the ephemeral mode workspace has no persistent storage attached. 
+All changes to the source code will be lost when the workspace is stopped unless they are pushed to a source code repository.
+This is done in order to improve the performance of a workspace startup and file system operations inside the IDE.
+User can disable ephemeral mode using the dedicated toggle during the workspace creation process from the user dashboard.
```

Comment:
We are now in an awkward situation, because as it is not specific to Hosted Che, this should be a reference to the user manual. But it is still missing there.

For more information about ephemeral mode, including how to disable it, see link:using-developer-environments-workspaces.html[Using developer environments - workspaces].

Reply:
toggle on UD on the workspace creation screen is hosted che specific and I have added src/main/images/hosted-che/ephemeral-toggle.png for clarification in the original PR

Reply:
Okay I still need to fix this last one.

Reply:
I have reworded it and moved to the creating... procedure the part explaining how to disable ephemeral mode. Is it correct like that @ibuziuk and @rkratky?

Reply:
works for me